app/src/org/odk/collect/android/views/media/MediaLayout.java

```diff
@@ -214,6 +214,7 @@ else if (qrCodeContent != null ) {
                         mImageView.setPadding(10, 10, 10, 10);
                         mImageView.setImageBitmap(b);
                         mImageView.setId(23423534);
+                        mImageView.setAdjustViewBounds(true);
```

Comment:
@amstone326 Can you confirm with whether this jives with your image resizing work? I'm honestly not sure how it overlaps.